mne/epochs.py

```diff
@@ -834,21 +835,7 @@ def _compute_aggregate(self, picks, mode='mean'):
 
         if self.preload:
             n_events = len(self.events)
-
-            if mode == "mean":
-                def fun(data):
-                    return np.mean(data, axis=0)
-            elif mode == "median":
-                def fun(data):
-                    return np.median(data, axis=0)
-            elif mode == "std":
-                def fun(data):
-                    return np.std(data, axis=0)
-            elif callable(mode):
-                fun = mode
-            else:
-                raise ValueError("mode must be mean, median, std, or callable"
-                                 ", got %s (type %s)." % (mode, type(mode)))
+            fun = _check_combine(mode, valid=('mean', 'median', 'std'))
```

Comment:
no need to provide these `kwargs` because they are the default, but it's okay to leave I guess